packages/wonder-blocks-testing/src/harness/adapters/__tests__/data.test.js

```diff
@@ -10,57 +10,57 @@ describe("WonderBlocksData.adapter", () => {
         const children = <div>CONTENT</div>;
 
         // Act
-        const wrapper = mount(Data.adapter(children, []), {
-            includeDefaultTestHarness: false,
-        });
+        render(Data.adapter(children, []));
 
         // Assert
-        expect(wrapper).toHaveHTML("<div>CONTENT</div>");
+        expect(screen.getByText("CONTENT")).toBeInTheDocument();
     });
 
-    it("should render children within InterceptRequests when dataIntercepts configured", () => {
+    it("should render children within InterceptRequests when dataIntercepts configured", async () => {
```

Comment:
**suggestion:** Probably need to change the test name to something like - `should supports request interception via configured dataIntercepts`